scripts/docker-cmd.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+# fancy output
+green='\e[0;32m'
+red='\e[0;31m'
+yellow='\e[1;33m'
+NC='\e[0m' # No Color
+
+SPHINXINDEX_VOLUME="/var/lib/sphinxsearch/data/index/"
+SPHINXINDEX_EFS="/var/lib/sphinxsearch/data/index_efs/"
+SPHINXCONFIG="/etc/sphinxsearch/sphinx.conf"
+
+# index arrays config, filesystem and orphaned
+mapfile -t array_config < <(grep -E "^[^#]+ path" "${SPHINXCONFIG}" | awk -F"=" '{print $2}' | sed -n -e 's|^.*/||p')
+mapfile -t array_file < <(find "${SPHINXINDEX_EFS}" -maxdepth 1 -name "*.spd" | sed 's|.spd$||g' | sed -n -e 's|^.*/||p' )
+mapfile -t array_orphaned < <(comm -13 --nocheck-order <(printf '%s\n' "${array_config[@]}" | LC_ALL=C sort) <(printf '%s\n' "${array_file[@]}" | LC_ALL=C sort))
+
+# remove orphaned indexes
+echo -e "${green}looking for orphaned indexes in filesystem. ${NC}"
+for index in "${array_orphaned[@]}"; do
+    # skip empty elements
+    [[ -z ${index} ]] && continue
+    # skip .new files, we need them to sighup searchd / rotate index updates
+    if [[ ! $index == *.new ]]; then
+        echo -e "\t${red} deleting orphaned index ${index} from filesystem. ${NC}"
+        rm -rf "${SPHINXINDEX_EFS}${index}".*
+    fi
```

Comment:
I would not mix coding style and use also 
```bash
if [[ -z "${index}" ]]; then
  continue
fi
```
Also use `"` in both `if` with `${index}` and be consistent with `${index}` or `$index`